python/hail/plot/plots.py

```diff
@@ -317,7 +365,7 @@ def get_contig_index(x, starts):
             num_deleted += 1
 
     p = scatter(x, y, label=label, title=title, xlabel='Chromosome', ylabel='P-value (-log10 scale)',
-                size=size, legend=False, source_fields=hover_fields)
+                size=size, legend=False, source_fields=hover_fields, collect_all=True)
```

Comment:
No need to pass collect_all=True since it is ignored since x, y are Python objects.